### DISCUSSION TITLE
Raspberry Pi 5 not defined in native.py

### DISCUSSION THREAD

**Discussion by Uvis-Acurable** [gpiozero/gpiozero#1161] [Q&A]
Operating system: Raspbian bookworm
Python version: 3.11.2
Pi model: Pi 5
GPIO Zero version: 2.0.1
default pin factory: LGPIOFactory

Device.pin_factory = NativeFactory() throws 
2 exceptions: permissionError when trying to open /dev/mem and IOError as it's trying to open /dev/gpiomem

As I can see gpiomem does not exist in RPi5 however there are gpiomem0 to gpiomem4

When not using pin_factory works as expected

**Comment by bennuttall** [comment]
Native factory doesn't work on Pi 5. There is no `gpiomem` and you can't access `/dev/mem` either. The only supported pin factory is lgpio.

**Reply by Uvis-Acurable** [reply]
OK. Thanks for clarification
